config/default.yml

```diff
@@ -1884,7 +1884,7 @@ Metrics/AbcSize:
   # The ABC size is a calculated magnitude, so this number can be an Integer or
   # a Float.
   IgnoredMethods: []
-  Max: 15
+  Max: 16.5
```

Comment:
I was thinking that since the max is pretty arbitrary anyway, you could choose a number that will cause very few new offenses for users, i.e. a hight number. But I guess you gave it a lot of thought.

Reply:
I just tried a few settings. 17 might be better and trigger less new failures (which could be frustrating) for people using our default.

Reply:
First of all, thank you for the research and discussion to difficult default value.

> 17 might be better and trigger less new failures (which could be frustrating) for people using our default.

IMHO, I often hear talk about disabling `Metrics/AbcSize` cop or increasing the maximum value. I think it is better to have `Metrics/AbcSize` cop work with increasing the default maximum value, rather than disabling the cop due to strict default. 
So, if possible default value 17 seems to be a good value to me. (Of course, well thought out current value 16.5 is fine :-)

Reply:
Sure, bumped to 17